umap/static/umap/map.css

```diff
@@ -1497,6 +1496,9 @@ a.add-datalayer:hover,
         width: 35px;
         min-width: initial;
     }
+    .umap-main-edit-toolbox .map-name {
+        min-width: 80px;
```

Comment:
I don"t get that `min-width`. I'm expecting that the title will reduce before we lose the buttons, like in this example:

![image](https://github.com/umap-project/umap/assets/146023/50227690-097a-4cb5-b758-036262c8b733)

In master:

![image](https://github.com/umap-project/umap/assets/146023/bcea63a4-e3d4-450a-9eeb-d6b8d02232e1)